dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -56,9 +56,9 @@
   Thank you,
   %br
   - if @workshop.course == Pd::Workshop::COURSE_CSF
-    Dave Frye
+    Sam Morris
```

Comment:
Can we just standardize to the workshop organizers name? Do we have workshops without workshop organizers?

Reply:
Checked on production and it doesn't look like it:

```
[production] dashboard > Pd::Workshop.where(organizer_id: nil).count
=> 0
```

I can just remove the conditional